.pre-commit-config.yaml

```diff
@@ -569,31 +569,6 @@ repos:
         entry: ./scripts/ci/pre_commit/pre_commit_sync_init_decorator.py
         pass_filenames: false
         files: ^airflow/models/dag\.py$|^airflow/(?:decorators|utils)/task_group\.py$
-      - id: check-base-operator-usage
```

Comment:
I’m not sure if this should be removed outright. Instead we may want to check the two classes are imported from their respective modules.

Reply:
Just for clarify which behavior we expect here

**Option 1:**
Outside of core  `airflow.models.baseoperator.BaseOperator` and   `airflow.models.baseoperatorlink.BaseOperatorLink` should be imported from `airflow.models` 

**Option 2:**
Allow to use `airflow.models.baseoperator.BaseOperator` and `airflow.models.baseoperatorlink.BaseOperatorLink`
Ban import `airflow.models.baseoperator.BaseOperatorLink`

Reply:
In core: Allow only `airflow.models.baseoperator.BaseOperator` and `airflow.models.baseoperatorlink.BaseOperatorLink`

Outside core: Allow only `airflow.models.BaseOperator` and `airflow.models.BaseOperatorLink`

Reply:
yeah.